web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -543,35 +582,36 @@ div#main-content {
 #block-cj-material-local-actions nav .button.button-action,
 input.button {
   margin-top: 1em;
+  margin-left: 0;
   border: 3px solid transparent;
   border-radius: 2px;
   color: #FFFFFF;
   background: #1E5168;
   cursor: pointer;
   display: inline-block;
-  font-size: 16px;
+  font-size: 13px;
   font-weight: 400;
   letter-spacing: .5px;
-  line-height: 20px;
-  min-height: 54px;
-  height: initial;
-  min-width: 100px;
-  padding: 15px 24px;
+  //line-height: 20px;
+  //min-height: 54px;
+  //height: initial;
+  //min-width: 100px;
+  //padding: 15px 24px;
   text-align: center;
   text-decoration: none;
   text-transform: uppercase;
   transition: color .15s cubic-bezier(.4,0,.2,1),background-color .15s cubic-bezier(.4,0,.2,1);
   will-change: background-color;
 }
 .form-actions input.button {
-  margin-top: 50px;
+  //margin-top: 50px;
 }
```

Comment:
delete the rule rather than comment out.

Reply:
Done.